### PR TITLE
SSB: Add HiMyNamesL, fix typo in HoeenHero

### PR DESCRIPTION
**Pull request by CheeseMuffin**

### CODE REVIEW

File: mods/seasonal/moves.js [hunk 1]
```diff
@@ -197,6 +197,31 @@ exports.BattleMovedex = {
...
+	// HiMyNamesL
+	yallarestupid: {
+		accuracy: 100,
+		category: 'Status',
+		shortDesc: 'Confuse opponent, +1 spa, -1 opponent acc',
+		id: 'yallarestupid',
+		isNonstandard: true,
+		name: "Y'all are stupid!",
+		pp:10,
+		flags: {mirror: 1, protect: 1, reflectable: 1},
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js]: this seems snatchable

File: mods/seasonal/statuses.js [hunk 2]
```diff
@@ -84,13 +84,28 @@ exports.BattleStatuses = {
 			this.add('c|@grimAuxiliatrix|∠( ᐛ 」∠)＿');
 		},
 	},
+	himynamesl: {
+		exists: true,
```
Comment (HoeenCoder) [inline mods/seasonal/statuses.js]: needs `nocopy: true,` so it dosent get copied by baton pass, ive set all the other ones and will push once i fix the set i'm working on,

File: mods/seasonal/random-teams.js [hunk 3]
```diff
@@ -49,6 +49,12 @@ class RandomStaffBrosTeams extends RandomTeams {
 				signatureMove: 'Cha Cha Slide',
 				evs: {hp: 252, def: 16, spd: 240}, nature: 'Impish',
 			},
+			'HimMyNamesL': {
+				species: 'Xurkitree', ability: 'Beast Boost', item: 'Life Orb', gender: 'M', // ask gender
```
Comment (DeathlyPlays) [inline mods/seasonal/random-teams.js:53]: He's a Male dw

File: mods/seasonal/moves.js [hunk 4]
```diff
@@ -197,6 +197,31 @@ exports.BattleMovedex = {
...
 	},
+	// HiMyNamesL
+	yallarestupid: {
+		accuracy: 100,
+		category: 'Status',
+		shortDesc: 'Confuse opponent, +1 spa, -1 opponent acc',
+		id: 'yallarestupid',
+		isNonstandard: true,
+		name: "Y'all are stupid!",
+		pp:10,
```
Comment (DeathlyPlays) [inline mods/seasonal/moves.js]: Space after pp: for cleanliness tbh

File: mods/seasonal/moves.js [hunk 5]
```diff
@@ -197,6 +197,31 @@ exports.BattleMovedex = {
...
+		flags: {mirror: 1, protect: 1, reflectable: 1},
+		volatileStatus: 'Confusion',
```
Comment (DeathlyPlays) [inline mods/seasonal/moves.js]: Typically they lowercase confusion

File: mods/seasonal/random-teams.js [hunk 6]
```diff
@@ -49,6 +49,12 @@ class RandomStaffBrosTeams extends RandomTeams {
...
+				moves: ['Tail Glow', 'Thunderbolt', 'Grass Knot'],
+				signatureMove: 'Y\'all are stupid!',
+				evs: {hp: 4, spa: 252, spe:252}, nature: 'Timid',
```
Comment (DeathlyPlays) [inline mods/seasonal/random-teams.js]: For consistency space after spe:

**Review by DeathlyPlays — COMMENTED** [review]
Nitpicky changes tbh

File: mods/seasonal/moves.js (hunk 5 above)
Comment (HoeenCoder) [reply mods/seasonal/moves.js]: good catch, needs to be lower case or it wont work iirc

File: mods/seasonal/random-teams.js (hunk 3 above)
Comment (HoeenCoder) [reply mods/seasonal/random-teams.js:53]: we ask anyways

File: mods/seasonal/moves.js (hunk 5 above)
Comment (DeathlyPlays) [reply mods/seasonal/moves.js]: Most likely

**Pull request closed without merging**